### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by enahum**
@hmhealey sentry still needs to be updated

### CODE REVIEW

File: app/components/custom_list/index.disable_test.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+//         );
+//
+//         // should return null
+//         expect(wrapper.instance().renderFooter()).toMatchSnapshot();
+//
+//         // should return element
+//         wrapper.setProps({loading: true});
+//         expect(wrapper.instance().renderFooter()).toMatchSnapshot();
+//     });
+// });
```
Comment (saturninoabril) [inline app/components/custom_list/index.disable_test.js]: 👍

**Comment by hmhealey** [comment]
Does Sentry need something special to upgrade it, or did you just leave it because I usually handle that? Regardless, I'll make a ticket so that I remember to do that

**Comment by enahum** [comment]
@hmhealey I just left it cause you normally handle that and I don't really know if there is anything special that needs to be done or not

**Comment by hmhealey** [comment]
Cool. I just wanted to know if I should be setting aside an afternoon to debug stupid dependency upgrade errors.

I probably should do that anyway since they always hit when you least expect them 😛

**Pull request merged**